src/i18n/po/pt.po

```diff
@@ -408,7 +412,7 @@ msgid ""
 "technical users, command line interface (CLI) wallets are also available."
 msgstr ""
 "A Decred apresenta carteiras acessíveis através de interfaces gráficas de "
-"utilizador (GUIs) nos principais sistemas operativos (Windows, macOS, e "
+"utilizador (GUIs) nos principais sistemas operacionais (Windows, macOS, e "
```

Comment:
That does not look right. AFAIK it is sistemas operativos.

Reply:
Actually in Brazilian portuguese it is right "sistemas operacionais"